chpl/chpl-service/src/main/java/gov/healthit/chpl/job/RunnableJob.java

```diff
@@ -95,6 +98,9 @@ protected void addJobMessage(String message) {
      * job execution.
      */
     @Transactional
+    @CacheEvict(value = {
```

Comment:
Not sure this is the right place for this. I think it's only applicable to the MeaningfulUseUploadJob.

Reply:
Hmm, me neither. I traced the executing to hear at some point. The evict annotation not only evicts the cache, but it is the mechanism to send a message to the other nodes. So, maybe this is here because both things are needed - the cache is evicted and the other nodes need a message to evict also?